pkgs/applications/networking/cluster/kubectl-neat/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub, pkgs }:
+
+buildGoModule rec {
+  pname = "kubectl-neat";
+  version = "2.0.3";
+
+  src = fetchFromGitHub {
+    owner = "itaysk";
+    repo = "kubectl-neat";
+    rev = "v${version}";
+    hash = "sha256-j8v0zJDBqHzmLamIZPW9UvMe9bv/m3JUQKY+wsgMTFk=";
+  };
+
+  vendorHash = "sha256-vGXoYR0DT9V1BD/FN/4szOal0clsLlqReTFkAd2beMw=";
+
+  nativeBuildInputs = [
+    pkgs.bash
+  ];
```

Comment:
Why is this necessary?

Reply:
But I've just pushed the code without it - I checked that it's not required, but to be honest I'm not sure why

Reply:
I'll expect `bash` to be part of `stdenv`, and `buildGo121Module` to be based on `stdenv.mkDerivation`.

Reply:
Ah, okay. I see that there's a lot of stuff I need to learn about Nix :smile: 
Thanks for the review and additional information!